test/integration/targets/callback_default/runme.sh

```diff
@@ -125,6 +125,14 @@ export ANSIBLE_CHECK_MODE_MARKERS=0
 
 run_test default
 
+# Check for async output
+# NOTE: regex to match 1 or more digits works for both BSD and GNU grep
+ansible-playbook -i inventory test_async.yml > async_test.out
+cat async_test.out
```

Comment:
Minor optimization for capturing and displaying output.
```suggestion
ansible-playbook -i inventory test_async.yml 2>&1 | tee async_test.out
```